templates/projects/task_notification.html

```diff
@@ -5,16 +5,66 @@
 {% from "_formhelpers.html" import render_field %}
 
 {% block projectcontent %}
-<h3>Send email notification on the progress of tasks</h3>
+<h3>Send notification on the progress of tasks</h3>
 <form id="task_notification" method="post" action="{{ url_for('project.task_notification', short_name=project.short_name) }}">
     {{ form.hidden_tag() }}
     <fieldset>
         {{ render_field(form.remaining, placeholder= _('10')) }}
+        <div id="notif-options">
+            <p>{{_('The following options are available:')}}</p>
+            <ul>
+                <li><strong>{{_('Send email notification to project owner(s)')}}</strong>:
+                    {{_('The platform will send email when there are less than or equal to target-remaining tasks left')}}
+                </li>
+                <li><strong>{{_('POST To Provided Webhook')}}</strong> (Optional):
+                    {{_('The platform will hit your webhook with data')}}
+                    <a id="show-code">See how do we post data to your webhook</a>
+                    <pre id="code-example" hide><code>
+data=dict(project_id=project_id,
+          project_name=project_name,
+          remianing_tasks=the_number_of_remaining_tasks,
+          target_remaining=the_number_you_entered_above)
```

Comment:
Seems the form contents here could be rearranged. One of the option could be following, but can confirm with product owner on expectations from this feature

## Task Notification
### Send notification on the progress of the task
#### Notify project owner(s) via email when number of remaining tasks is less than or equal to
#### Webhook(Optional). 
Additional mechanism to be notified